src/home.js

```diff
@@ -1,21 +1,28 @@
-const homePage = () => {
-    const holder = document.getElementById('holder')
-    holder.setAttribute('id', 'holder')
+
+export default const Home = () => { 
+    holder.setAttribute('id', 'holder')// ?? where is this var coming from?
+    
     const main = document.createElement('div')
-    main.setAttribute('class', 'main')
     const message = document.createElement('p')
-    message.setAttribute('class', 'message')
-    message.innerText = 'Your local fan favorite!'
     const image = document.createElement('img')
-    image.src = 'https://www.idlememe.com/wp-content/uploads/2021/10/salt-bae-meme-idlememe.jpg'
     const message2 = document.createElement('p')
+    
+    //Set attributes
+    main.setAttribute('class', 'main')
+    message.setAttribute('class', 'message')
     message2.setAttribute('class', 'message')
+    
+    //Set contents
+    image.src = 'https://www.idlememe.com/wp-content/uploads/2021/10/salt-bae-meme-idlememe.jpg'
+    message.innerText = 'Your local fan favorite!'
     message2.innerText = 'Serving you since 1890'
 
+    //A strectch, but you could just append Home to this in index.js.
+    // This works though
     holder.appendChild(main) 
```

Comment:
**Future-proofing thought (Not a critical problem):** 

Remove the holder.append statement(?) 

This component should not handle its own appending to any element. The code that uses this component should have control over this. So index.js will get this component, and handle appending it to the body. This allows for a more diverse use case for this component, you could append it directly to the body, to another element, and so on.